Bixal/move.mil
# Add json file conv

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request adds a custom module containing a controller that:

- Looks for 'household_goods_weights.json' in 'lib/data/json-data'
- Reads the file or returns an error if the file is not found
- Converts the object
- Returns the desired JsonObject
 
## Testing

1. Pull changes locally
2. Install module
3. Navigate to 'json_response/weight_calculator' to see the printed JsonObject

## Screenshots

_Attach relevant before and after screenshots here._
